platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/AttributeSharingSessionDataStore.java

```diff
@@ -55,7 +55,7 @@ public void updateSessionAttributes(String id, Map<String, Object> sessionAttrib
           "Storing new attributes for session {} at context {}",
           id,
           _context.getCanonicalContextPath());
-      synchronized (this) {
+      synchronized (sessionDataMap) {
```

Comment:
I just realized I need to make this change in the other 2 synchronized blocks in this class.